tests/unit/testUncaught.js

```diff
@@ -50,6 +52,13 @@ test('Uncaught handler setup', function (t) {
 });
 
 test('Test uncaught shutdown behavior', function (t) {
+  if (!isString(process.env.GOOGLE_APPLICATION_CREDENTIALS)
```

Comment:
Ok, I'm fine leaving this in for now. It would be good to avoid if possible in the future as we have had skipped tests start failing unnoticed in the past.